capstone_project/src/main/webapp/images/reactions/emoji_u1f602.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.1.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
+<g id="facial_expressions">
```

Comment:
nit: not sure if svg tabbing is different, but I am seeing different tabbing styles! @line 5, not tabbed, @line 7 is  ~8?

Reply:
I'm not sure, these were generated by Adobe Illustrator as noted in the comment. These are [Google's emojis](https://github.com/googlefonts/noto-emoji)

Reply:
Okay!